consensus/quorum/one-node-staked-vote.go

```diff
@@ -13,14 +13,23 @@ import (
 )
 
 var (
-	twoThirds = numeric.NewDec(2).QuoInt64(3)
-	hSentinel = numeric.ZeroDec()
+	one   = numeric.OneDec()
+	two   = numeric.NewDec(2)
+	three = numeric.NewDec(3)
+
+	twoThirds         = two.Quo(three)
+	ninetyPercent     = numeric.MustNewDecFromStr("0.90")
+	sixtyEightPercent = numeric.MustNewDecFromStr("0.68")
+	thirtyTwoPercent  = numeric.MustNewDecFromStr("0.32")
+	quorumThreshold   = sixtyEightPercent.Mul(numeric.NewDec(100)).RoundInt()
```

Comment:
quorum threshold should be 1/3 in Dec. basically 0.6666666666666666666..  And the condition of quorum is > 1/3